### ISSUE TITLE
No vertical scroll bar in table when using sticky header

### ISSUE THREAD

**Issue by alqu** [buefy/buefy#2343]
### Overview of the problem

**Buefy** version: [0.8.13]
**Vuejs** version: [2.6.11]
**OS/Browser**: MacOS / Chrome 80.0

### Description

The vertical scrollbar is not visible when using the sticky headers feature on b-table.

### Steps to reproduce

1. Use Chrome 80.0
2. go to https://buefy.org/documentation/table/#sticky-headers-and-columns

### Expected behavior

You should see a vertical scrollbar like in Firefox or Safari

### Actual behavior

There is no scrollbar visible.
Nevertheless, the table is rendered properly, only the scrollbar is missing.

**Comment by jtommy** [comment]
@alqu i checked it and there is no issue, probably your os hide scrollbar (ios should have this option)

**Comment by alqu** [comment]
Hm, I checked on my **Mac** w/ Catalina, not iOS. And the problem exists in Chrome - see screenshot:
![no Scrollbars in Chrome w Buefy](https://user-images.githubusercontent.com/12250845/78017904-46757400-734d-11ea-84c5-12f22bafce0d.png)

After re-installing Chrome (and deleting my profile) it still does not work.
But the scrollbars work flawlessly on Bulma's website - so I assume that there is something in your css:
![Scrollbars on Bulma WS](https://user-images.githubusercontent.com/12250845/78017999-6ad15080-734d-11ea-811b-5a546368b4ca.png)
(I did a screenshot a tad too late, so the vertical scrollbar already faded out)

To make sure this is not a "broken on my machine"-problem I checked on another Mac w/ Chrome, and there it works. Really strange.

I hope this information helps to point into the right direction.
